medusa/tv/series.py

```diff
@@ -1752,11 +1757,10 @@ def refresh_dir(self):
                     with cur_ep.lock:
                         # if it used to have a file associated with it and it doesn't anymore then
                         # set it to app.EP_DEFAULT_DELETED_STATUS
-                        if cur_ep.location and cur_ep.status in Quality.DOWNLOADED:
+                        if cur_ep.location and cur_ep.status in DOWNLOADED:
```

Comment:
Since `Quality.DOWNLOADED` was a list, I think this changes the logic. So use:
```py
if cur_ep.location and cur_ep.status == DOWNLOADED:
# or
if cur_ep.location and cur_ep.status in (DOWNLOADED, ):
# or
if cur_ep.location and cur_ep.status in [DOWNLOADED]:
```